### DISCUSSION TITLE
Good afternoon, I noticed my github has been flagged without any prior notifications and for what exactly..This has caused me some reduction in my project scores and which is affecting my total score. I am really not happy about this and want this to be resolved immediately. CHARLESUKE is my github name with this emails.

### DISCUSSION THREAD

**Discussion by CHARLESUKE** [community/community#127098] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Good afternoon, I noticed my github has been flagged without any prior notifications and for what exactly..This has caused me some reduction in my project scores and which is affecting my total score. I am really not happy about this and want this to be resolved immediately. CHARLESUKE is my github name 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ebndev** [comment]
Hi @CHARLESUKE,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. If you've already opened a ticket, it will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this, and any duplicate discussions. We appreciate your understanding!
